hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformations.scala

```diff
@@ -270,7 +270,7 @@ object DemographicsTransformations {
       if (allActivities.contains[Long](ActivityValues(activity))) {
         rawRecord.getOptionalNumber(s"dd_${activity}_m").orElse(Some(3L))
       } else {
-        None
+        rawRecord.getOptionalNumber(s"dd_${activity}_m").orElse(None)
```

Comment:
worth noting, when simplifying this, that calling `orElse(None)` on an `Option[X]` is redundant, since it'll already be `None` if the `orElse` gets used.

Reply:
Ok, reduced redundancy in this commit https://github.com/DataBiosphere/dog-aging-ingest/pull/121/commits/790e8361699b7d4970dd45fd031e918de5346b31